layers/+emacs/org/README.org

```diff
@@ -651,9 +651,12 @@ The evilified org agenda supports the following bindings:
 |----------------------+-----------------------------------|
 | ~M-SPC~ or ~s-M-SPC~ | org-agenda transient state        |
 | ~SPC m a~            | org-agenda                        |
+| ~SPC m ,~            | org-agenda-ctrl-c-ctrl-c          |
+| ~SPC m c~            | org-agenda-capture                |
 | ~SPC m C c~          | org-agenda-clock-cancel           |
 | ~SPC m C i~          | org-agenda-clock-in               |
 | ~SPC m C o~          | org-agenda-clock-out              |
+| ~SPC m C i~          | org-agenda-clock-goto             |
```

Comment:
This binding appears to be duplicate with `org-agenda-clock-in` above. Based on below, I think you meant to use a `j` here.

Reply:
You're absolutely right, thanks!